runtime/doc/usr_04.txt

```diff
@@ -122,6 +122,7 @@ single-letter command:
 	C  stands for  c$  (change to end of the line)
 	s  stands for  cl  (change one character)
 	S  stands for  cc  (change a whole line)
+	Y  stands for  y$  (yank/copy to end of the line)
```

Comment:
Not sure if we should use copy here, do we use copy anywhere else in the manual?

Reply:
Yes, further down on `*04.6*	Copying text`. I'm a bit conflicted as well, my reasoning was that since section 6 (Copying text) didn't assume prior knowledge of what "yank" meant, then maybe the text before shouldn't either.

Reply:
Hmmm, yeah, the inconsistency is bothering. I'm removing it.